model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -40,7 +40,7 @@ data class LicenseFinding(
     val location: TextLocation
 ) : Comparable<LicenseFinding> {
     companion object {
-        private val COMPARATOR = compareBy<LicenseFinding> { it.license.toString() }.thenBy(LicenseFinding::location)
+        private val COMPARATOR = compareBy(LicenseFinding::license).thenBy(LicenseFinding::location)
```

Comment:
Commit-message: does this commit actually change the serialization of the `scanner` bit in the `OrtResult` ?

Reply:
It should not, as the sorting should still be exactly the same. Just the code was refactored.